xml_inscriptions/transcribed/CA.Berk.UC.HMA.G.6-19905.xml

```diff
@@ -0,0 +1,165 @@
+                                </support>
+                                <condition ana="#fragment.single">
+                                    <p/>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="0" writtenLines="1">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <!--  *************************************  <handDesc>  ********************************* -->
+                        <handDesc hands="1">
+                            <handNote ana="#inscribed"> </handNote>
+                        </handDesc>
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p>inscribed wreath, above text</p>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+                    <!--  *************************************  <history>  *********************************  -->
+                    <history>
+                        <summary>
+                            <rs/>
+                            <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                        </summary>
+                        <origin>
+                            <date evidence="#unspecified" notBefore="-0031" notAfter="0641">31 BC to
+                                641 AD</date>
+                            <placeName ref="africa.egypt">Deir el-Ballas</placeName>
+                        </origin>
+                        <provenance type="transferred" subtype="donated">
+                            <p>Donated by <persName role="donor">P. A. Hearst</persName> to <persName role="recipient">Hearst Museum</persName></p>
+                        </provenance>
+                        <provenance type="transferred" subtype="acquired">
+                            <p>Collected by <persName>George Andrew Reisner</persName></p>
+                        </provenance>
+                        <provenance type="found" subtype="discovered">
+                            <p>Found in Room 171, Northern Kom (Northern Palace), <placeName>Deir el-Ballas, Egypt</placeName></p>
+                        </provenance>
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl"/>
```

Comment:
Make sure to include bibliography if you have it; if not, you can remove this bit (and we can always re-add it back in).